uint32 TagID
read_apriltags/rpy euler_rpy
geometry_msgs/Point32[4] corners_array
float32 tag_size
geometry_msgs/Point32 position_xyz

